docs/docs-beta/docs/dagster-plus/deployment/management/settings/customizing-agent-settings.md

```diff
@@ -4,8 +4,132 @@ sidebar_position: 80
 unlisted: true
 ---
 
-{/* TODO move from https://docs.dagster.io/dagster-plus/deployment/agents/customizing-configuration */}
+:::note
+This guide is applicable to Dagster+.
+:::
+
+{/* /deployment/dagster-instance */}
+The Dagster+ Agent is a special variant of the Dagster instance used in [Dagster Open Source](/todo.md) and is configured through the same `dagster.yaml` file. You can customize your agent with these settings.
+
+:::note
+{/* /dagster-plus/deployment/agents/kubernetes/configuring-running-kubernetes-agent */}
+For [Kubernetes agents](/todo.md) deployed with the Dagster+ Helm chart, you'll need to refer to the Helm chart's config map for customizing the agent.
+:::
+
+## Enabling user code server TTL
+
+User code servers support a configurable time-to-live (TTL). The agent will spin down any user code servers that haven’t served requests recently and will spin them back up the next time they’re needed. Configuring TTL can save compute cost because user code servers will spend less time sitting idle.
```

Comment:
```suggestion
User code servers support a configurable time-to-live (TTL). The agent will spin down any user code servers that haven't served requests recently and will spin them back up the next time they're needed. Configuring TTL can save compute cost because user code servers will spend less time sitting idle.
```